Sources/Filters/Sources/PlatonicSolidSource/index.d.ts

```diff
@@ -0,0 +1,97 @@
+import { vtkAlgorithm, vtkObject } from '../../../interfaces';
+import { DesiredOutputPrecision } from '../../../Common/DataModel/DataSetAttributes';
+import { SolidType } from './Constants';
+
+/**
+ *
+ */
+export interface IPlatonicSolidSourceInitialValues {
+  solidType?: SolidType;
+  outputPointsPrecision?: DesiredOutputPrecision;
+}
+
+type vtkPlatonicSolidSourceBase = vtkObject &
```

Comment:
why do you have to do something custom like that ? can't you simply derive from vtkObject and vtkAlgorithm ?

Reply:
ok to deal with that in a separate PR